src/Components/GEOSldas_GridComp/GEOSmetforce_GridComp/GEOS_MetforceGridComp.F90

```diff
@@ -13,12 +13,11 @@ module GEOS_MetforceGridCompMod
   use LDAS_ensdrv_Globals, only: logunit,logit   !,root_logit
   use LDAS_DateTimeMod, only: date_time_type, date_time_print
   use LDAS_TileCoordType, only: tile_coord_type
-  use LDAS_TileCoordType, only: T_TILECOORD_STATE 
   use LDAS_TileCoordType, only: TILECOORD_WRAP
   use LDAS_ForceMod, only: LDAS_GetForcing => get_forcing
   use LDAS_ForceMod, only: LDAS_move_new_force_to_old
   use LDAS_ForceMod, only: FileOpenedHash,GEOS_closefile
-  use LDAS_ForceMod, only: im_world_cs
+  use LDAS_ForceMod, only: im_world_cs, neighbor_offset
```

Comment:
maybe replace "neighbor_offset" with "get_neighbor_offset" to make it more clear where that this variable belongs to subroutine get_neighbor_index() ?